### PR TITLE
[OSDOCS-1451] - Specified Managing Security Context Constraints

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR adds the "Managing security context constraints" to the OSD book and removes sections that are not relevant.

JIRA: https://issues.redhat.com/browse/OSDOCS-1451

PREVIEW: https://deploy-preview-38236--osdocs.netlify.app/openshift-dedicated/latest/authentication/managing-security-context-constraints.html

### CODE REVIEW

File: authentication/managing-security-context-constraints.adoc [hunk 1]
```diff
@@ -12,7 +12,9 @@ include::modules/security-context-constraints-pre-allocated-values.adoc[leveloff
 
 include::modules/security-context-constraints-example.adoc[leveloffset=+1]
 
+ifndef::openshift-dedicated[]
```
Comment (maxwelldb) [inline authentication/managing-security-context-constraints.adoc:15]: This is not included in my dedicated preview builds. Isn't that expected behavior?

**Comment by EricPonvelle** [comment]
@wgordon17 - PTAL at this PR. I've pulled content from OCP, updated the OSD side, and I've hidden constraints/sections that you pointed out. Please let me know if you have any feedback on this PR.

**Comment by wgordon17** [comment]
Wow, the new navigation stuff looks terrible on this preview build! (cc @vikram-redhat).

But otherwise, the content looks good @EricPonvelle! 👍

**Comment by vikram-redhat** [comment]
> Wow, the new navigation stuff looks terrible on this preview build! (cc @vikram-redhat).

Esh sorry about that. Netlify for existing PRs (created before the new design went in) won't reload unless I delete and purge the cache.

**Comment by EricPonvelle** [comment]
@xueli181114 PTAL at this PR and let me know if everything is accurate. Thanks!

**Comment by EricPonvelle** [comment]
PR continued here: https://github.com/openshift/openshift-docs/pull/39137

**Pull request closed without merging**